models/offer.js

```diff
@@ -58,7 +58,8 @@ const offerSchema = new Schema(
   {
     timestamps: true,
     toJSON: { virtuals: true },
-    toObject: { virtuals: true }
+    toObject: { virtuals: true },
+    id: false
```

Comment:
what 'id: false' does?
if it removes duplicate 'id' field from the result, then add it everywhere?

Reply:
yes, it removes duplicated id field, but i'm not sure it's needed in schemas that we're not gonna populate